IntegrationTests/Runner/Scenarios/URLSession/URLSessionScenarios.swift

```diff
@@ -39,11 +39,6 @@ private class CompositedURLSessionDelegate: NSObject, URLSessionTaskDelegate, UR
     }
 }
 
-/// An example of instrumenting existing `URLSessionDelegate` with `DDURLSessionDelegate` through inheritance.
-private class CustomURLSessionDelegate: NSObject, URLSessionDataDelegate {
```

Comment:
Moving to its own file with `internal` accessor for re-use in stop-core-instance test.